main.go

```diff
@@ -151,6 +155,46 @@ func updateNDK(revision string) error {
 	return nil
 }
 
+func unWrapNDKDirIfNeeded(ndkHome string, revision string) error {
+	platformDirName := "platforms"
+	platformDir := filepath.Join(ndkHome, platformDirName)
+	p, err := pathutil.IsDirExists(platformDir)
+	if err != nil {
+		return fmt.Errorf("could not check if directory %s exists. Error: %v", platformDir, err)
+	}
+	if p {
+		return nil
+	}
+	revDirName := fmt.Sprintf("android-ndk-r%s", revision)
+	revDir := filepath.Join(ndkHome, revDirName)
+	r, err := pathutil.IsDirExists(revDir)
+	if err != nil {
+		return fmt.Errorf("could not check if directory %s exists. Error: %v", revDir, err)
```

Comment:
Just a note. We are using %s instead of %v for error printing, but the %v if perfectly fine.

Reply:
Changed to string.